sample-apps/00-fluentBit/kubernetes/fluentbit.yaml

```diff
@@ -92,15 +92,13 @@ data:
         Preserve_Key      On
   
   output-data-prepper.conf: |
-    [OUTPUT]
-        Name              http
-        Match             *
-        Host              data-prepper.data-prepper.svc.cluster.local
-        Port              2021
-        tls               Off
-        tls.verify        Off
-        Format            json
-        URI               /log/ingest
+      [OUTPUT]
+        Name                 opentelemetry
+        Match                *
+        Host                 otel-collector.otel-collector.svc.cluster.local
+        Port                 55681
+        tls                  off
```

Comment:
I recommend checking this setup using SSL enabled because many users follow it as a reference.

Reply:
I would like to ask you to create an issue for us to enable SSL.

Reply:
https://github.com/aws-samples/observability-with-amazon-opensearch/issues/62